src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+interface NavLinkClassName {
+  isActive: boolean;
+}
+
+export const Navigation: React.FC = () => {
+  const getClasses = ({ isActive }: NavLinkClassName) => classNames(
+    'navbar-item',
+    { 'is-active': isActive },
+  );
+
+  return (
+    <nav
+      className="navbar is-light is-fixed-top is-mobile has-shadow"
+      data-cy="Nav"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink to="/" className={getClasses}>
+            Home
+          </NavLink>
+
+          <NavLink to="/tabs" className={getClasses}>
+            Tabs
+          </NavLink>
```

Comment:
You could move this to a separate component, pass 'to' and 'text' params via props and then you dont need your helper function for classes. But I dont know if it`s so critical..